resources/zib/zib-HearingFunction.HearingAid.xml

```diff
@@ -35,8 +33,15 @@
   <differential>
     <element id="DeviceUseStatement">
       <path value="DeviceUseStatement" />
+      <short value="MedicalDevice / HearingAid" />
       <definition value="The medical aid used to help the patient hear." />
       <comment value="This is a specialization of the [zib-MedicalDevice](http://nictiz.nl/fhir/StructureDefinition/zib-MedicalDevice) profile to represent HearingAid devices within the context of zib HearingFunction." />
+      <alias value="HorenHulpmiddel::MedischHulpmiddel" />

```

Comment:
Should we use just 'HorenHulpmiddel' and add a known issue?

Reply:
See above

Reply:
I'm not sure what should be the correct way actually. The guidelines only specify how we should handle `.short`, not `.alias`. The zib compliance tool doesn't seem to care.

Reply:
Fixed ... no known issue necessary